tsconfig.json

```diff
@@ -43,6 +43,7 @@
     "next-env.d.ts",
     "**/*.ts",
     "**/*.tsx",
+    ".storybook/preview.tsx",
```

Comment:
Having renamed this file will cause major conflicts with #2111, but it's not a big file so I'm sure we'll figure it out when we get there.

Reply:
It turned out to not be so bad. git merge understood the file was renamed and there was only a minor conflict to resolve in `.storybook/preview`. See #2358

Reply:
Git, I underestimated you. 🙏